.gitignore

```diff
@@ -99,3 +99,14 @@ config.txt
 
 # Mac Finder
 .DS_Store
+
+# mattl uses Emacs
```

Comment:
foolip uses Emacs too

Reply:
FWIW, when I once tried to add `*~` to the .gitignore of a project where the maintainer wasn't an Emacs user, it was not so well received and I was pointed to something like https://stackoverflow.com/a/7335487

I actually haven't set that up myself, but that is a way to not being annoyed by this kind of thing in new repos.

Reply:
Yes, the correct answer is to set up a global `gitignore` file (there are default paths, you don't actually need to set any config path) containing OS and editor-specific ignores (and GitHub's gitignores repo includes standard ones for both).